knative-kubecon/README.md

```diff
@@ -230,4 +230,132 @@ $ curl -H "Host: helloworld-openshift.myproject.example.com" "http://$(minishift
 
 It's alive!
 
-**To be continued...**
\ No newline at end of file
+## 3. The Eventing Components
+
+The previous steps showed you how to invoke a Knative Serving application from an HTTP request, in that case submitted via curl.
+Knative Eventing is all about how you can invoke those functions in response to other events such as those received from message brokers or external applications.
+In this part of the demo we are going to show how you can receive Kubernetes platform events and route those to a Knative Serving application.
+
+Knative Eventing is built on top of three primitives:
+* Event Sources
+* Channels
+* Subscriptions
+
+_Event Sources_ are the components that receive the external events and forward them onto _Channels_.
```

Comment:
Nit: I made all new words **bold** above vs _italic_. I think we should be consistent with either way.

Reply:
I've adopted your style